### ISSUE TITLE
How to set a perspective camera?

### ISSUE THREAD

**Issue by boycehbz** [facebookresearch/eft#13]
Hi, I want to render SMPL mesh with perspective projection. How to set a perspective camera?

**Comment by jhugestar** [comment]
It is quite tricky since the predicted  camera parameters are based on weak-perspective camera model. This might be possible if you consider a fixed intrinsic parameters by which you may convert the scaling factor of weak-perspective to depth. One tricky part, however, is that in perspective projection you also need to consider the camera center. It can be the bounding box center or image center. In weak-perspective camera, it doesn't not matter, but it matters in perspective, depending on your application.  

In short, we do not provide a solution for this. Sorry about that.
